incubator/harbor/templates/clair/clair-dpl.yaml

```diff
@@ -0,0 +1,37 @@
+{{ if .Values.clair.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "harbor.fullname" . }}-clair
+  labels:
+{{ include "helm.labels" . | indent 4 }}
+    component: clair
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+{{ include "helm.labels" . | indent 8 }}
+        component: clair
+    spec:
+      containers:
+      - name: clair
+        image: vmware/clair:v2.0.1-photon
```

Comment:
The clair image should have a configurable image. I understand from reading elsewhere that this isn't fully baked and working yet.